### PR TITLE
Add support for passing os partition size when creating nodes

### PR DESCRIPTION
**Pull request by Andrius521**
#### Which component this PR applies to?
cluster-autoscaler

#### What type of PR is this?
/kind feature

#### What this PR does / why we need it:
Set OS partition size for nodes 
#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cluster-autoscaler/cloudprovider/cherryservers/README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ In the above file you can modify the following fields:
...
 | cluster-autoscaler-cloud-config | Global/reservation      | The values "require" or "prefer" will request the next available hardware reservation for new servers in selected region & plan. If no hardware reservations match, "require" will trigger a failure, while "prefer" will launch on-demand servers instead (default: none)  |
 | cluster-autoscaler-cloud-config | Global/hostname-pattern | The pattern for the names of new Cherry Servers servers (default: "k8s-{{.ClusterName}}-{{.NodeGroup}}-{{.RandString8}}" )                  |
+| cluster-autoscaler-cloud-config | Global/os-partition-size | The OS partition size in gigabytes for new nodes in the nodepoll (eg: `60`)                 |
```
Comment (deitch) [inline cluster-autoscaler/cloudprovider/cherryservers/README.md:32]: This should be `nodepool` instead of `nodepoll`.

Comment (deitch) [reply cluster-autoscaler/cloudprovider/cherryservers/README.md:32]: Also, should describe the default.

**Review by deitch — CHANGES_REQUESTED** [review]
Just a documentation typo.

File: cluster-autoscaler/cloudprovider/cherryservers/README.md (hunk 1 above)
Comment (Andrius521) [reply cluster-autoscaler/cloudprovider/cherryservers/README.md]: Thanks, made a fix.

**Review by deitch — APPROVED** [review]
LGTM.

As an aside, what does the "default 'none'" mean? Does it get no partition at all, OS entirely in memory? Or that it defaults to whatever the cherryservers default is?

**Comment by Andrius521** [comment]
> LGTM.
> 
> As an aside, what does the "default 'none'" mean? Does it get no partition at all, OS entirely in memory? Or that it defaults to whatever the cherryservers default is?

By default OS root directory will be given full disk size.

**Pull request merged**